pkg/builder/builder.go

```diff
@@ -61,6 +61,12 @@ type Options struct {
 	PubKeyPath string
 	// OutputOpts is the output options.
 	OutputOpts string
+	// ExportCache is the options to export cache.
+	// e.g. type=registry,ref=docker.io/username/image
+	ExportCache string
+	// ImportCache is the options to import cache.
+	// e.g. type=registry,ref=docker.io/username/image
+	ImportCache string
```

Comment:
```suggestion
	// ExportCache is the option to export cache.
	// e.g. type=registry,ref=docker.io/username/image
	ExportCache string
	// ImportCache is the option to import cache.
	// e.g. type=registry,ref=docker.io/username/image
	ImportCache string
```

Also the previous one: "OutputOpts is the output option."